sdk/arm/storage/2019-06-01/armstorage/README.md

```diff
@@ -0,0 +1,48 @@
+# Azure Storage Module for Go
+
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/Azure/azure-sdk-for-go/sdk/arm/storage/2019-06-01/armstorage)](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/arm/storage/2019-06-01/armstorage)
+
+The `armstorage` module provides operations for working with Azure storage resources.
+
+[Source code](https://github.com/Azure/azure-sdk-for-go/tree/master/sdk/arm/storage/2019-06-01/armstorage)
+
+# Getting started
+
+## Install the package
+
+This project uses [Go modules](https://github.com/golang/go/wiki/Modules) for versioning and dependency management.
+
+Install the Azure Storage module:
+
+```sh
+go get -u github.com/Azure/azure-sdk-for-go/sdk/arm/storage/2019-06-01/armstorage
+```
+
+## Prerequisites
+
+- an [Azure subscription](https://azure.microsoft.com/free/)
+- Go 1.13 or above
+
+## Provide Feedback
```

Comment:
We need to have a section for usage pattern (e.g. creating a client and making a call to the service)

Reply:
That's in the samples.  I explicitly didn't want to duplicate it in the README.